MarsFramework/Pages/ProfilePage.cs

```diff
@@ -1,37 +1,106 @@
+            PageFactory.InitElements(Global.GlobalDefinitions.Driver, this);
+        }
+
+        //Click on Dropdown link
+        [FindsBy(How = How.XPath, Using = "//span[contains(@class, 'item ui dropdown link')]")]
+        private IWebElement DropDownLink { get; set; }
+
+        //Go to Profile Page
+        [FindsBy(How = How.XPath, Using = "//a[text()[contains(.,'Go to Profile')]]")]
+        private IWebElement GotoProfile { get; set; }
+
+        // Click on Description Write Icon
+        [FindsBy(How = How.XPath, Using = "//i[@class = 'outline write icon']")]
+        private IWebElement DescriptionWriteIcon { get; set; }
+
+        //Enter Description
+        [FindsBy(How = How.XPath, Using = "//form/div/div/div[2]/div[1]/textarea")]
+        private IWebElement Description { get; set; }
+
+        //Save Description
+        [FindsBy(How = How.XPath, Using = "//form/div/div/div[2]/button")]
+        private IWebElement SaveDescription { get; set; }
+
+        //Select Language tab
+        [FindsBy(How = How.XPath, Using = "//a[text()[contains(. , 'Languages')]]")]
+        private IWebElement LanguageTab { get; set; }
+
+        //Click on Language AddNew Button
+        [FindsBy(How = How.XPath, Using = "(//div[contains(text(),'Add New')])[1]")]
+        private IWebElement LanguageAddNewButton { get; set; }
+
+        //Enter a New Language
+        [FindsBy(How = How.XPath, Using = "//input[@placeholder = 'Add Language']")]
+        private IWebElement AddLanguageTextBox { get; set; }
+
+        //Click on Language Level Dropdown
+        [FindsBy(How = How.XPath, Using = "//select[@class = 'ui dropdown'][@name = 'level']")]
+        private IWebElement LanguageLevelDropDown { get; set; }
+
+        //Click on Add Button
+        [FindsBy(How = How.XPath, Using = "//input[@type = 'button'][@value = 'Add']")]
+        private IWebElement AddLanguageButton { get; set; }
+
+        //Click on language edit write icon
+        [FindsBy(How = How.XPath, Using = "(//i[@class = 'outline write icon'])[2]")]
+        private IWebElement LanguageEditWriteIcon { get; set; }
+
+        // Enter Edited Language
+        [FindsBy(How = How.XPath, Using = "//input[@placeholder = 'Add Language']")]
+        private IWebElement EditLanguageTextBox { get; set; }
+
+        // Click on Edit Language Dropdown
+        [FindsBy(How = How.XPath, Using = "//select[@class = 'ui dropdown'][@name = 'level']")]
+        private IWebElement EditLanguageLevelDropDown { get; set; }
 
+        //Click on Update Button
+        [FindsBy(How = How.XPath, Using = "(//input[@type = 'button'][@value = 'Update'])[1]")]
+        private IWebElement LanguageUpdateButton { get; set; }
+
+        //Click on Delete Icon
+        [FindsBy(How = How.XPath, Using = "(//i[@class = 'remove icon'])[1]")]
+        private IWebElement DeleteLanguage { get; set; }
+
+        private void GoToLanguageTab(IWebDriver Driver)
```

Comment:
avoid passing the driver, if you need this then make it available as a instance variable

Reply:
Thanks & fixed it